lib/instrumentation/modules/_lambda-handler.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+const { getLambdaHandler } = require('../../lambda')
+module.exports = function (module, agent, { version, enabled }) {
```

Comment:
It would be nice to support `enabled` so a user could put "lambda" in `disableInstrumentations`.

Reply:
It was quick so I added the enabled check in cdbaa1d696e8974f7d67b9c58e59981139c8feab no harm in having it there regardless of which direction we head. As for supporting using the identifier "lambda" in `disableInstrumentations` -- I'm not sure I understand what you're proposing. 

Are you suggesting we special case `lambda` as an identifier for `disableInstrumentations` and use it to determine what to pass to this handler function?  If so I'm open to that but there's a real module named lambda: https://www.npmjs.com/package/lambda -- doubtful we'd ever instrument it but still feels a little weird to use its identifier.  WDYT?

If that's not what you're suggesting -- then what are you suggesting? ;)

Reply:
> Are you suggesting we special case `lambda` as an identifier for `disableInstrumentations` and use it to determine what to pass to this handler function?

That *had* been what I was suggesting, but I hadn't thought it through. I agree with you that this would be weird. I take back that "it would be nice to" suggestion.